.github/workflows/build_debug_apk.yml

```diff
@@ -93,3 +93,15 @@ jobs:
           ARTIFACT_DIRECTORY: artifacts/debug-apk
           GITHUB_REF: ${{ github.ref }}
           ACTION: internal_notifications
+
+  crowdin-auto-merge:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.user.login == 'lawnchair-bot' && github.event.pull_request.title == 'New Crowdin translations by Github Action'
+    needs: build-debug-apk
+    steps:
+      - env:
+          GH_TOKEN: ${{ secrets.LAWNCHAIR_BOT_TOKEN }}
+          PR_URL: ${{ github.event.pull_request.html_url }}
+        run: |
+          gh pr comment $PR_URL --body "No issues found, auto-merging the Crowdin pull request."
+          gh pr merge $PR_URL --auto --squash --delete-branch
```

Comment:
Adding `--auto` flag to ensure auto merges works after we enabled auto merge status check in repo settings.